composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "spryker-sdk/spryk",
     "type": "library",
-    "description": "Spryk module",
+    "description": "Spryk library",
```

Comment:
We might have to keep this and the readme h1
Since this was one, so for releasing this might have to stay as such